recipes/datreant/meta.yaml

```diff
@@ -0,0 +1,26 @@
+package:
+  name: datreant
+  version: 1.0
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - datreant
+
+about:
+  home: http://datreant.org/
+  license: BSD 3-Clause
+  summary: "datreant namespace package"
+
+extra:
+  recipe-maintainers:
+    - datreant/coredevs
```

Comment:
Ah, so, I don't think this would work. Though I'm actually a little unsure. It is better to list explicitly when members are maintainers by their handles.